README.md

```diff
@@ -161,6 +162,60 @@ docker run --privileged -v /var/run/docker.sock:/var/run/docker.sock ternd --dri
 
 You can make this change to the `docker_run.sh` script to make it easier.
 
+## Creating a Kubernetes Job<a name="k8s-deployment">
```

Comment:
Ack, sorry I didn't catch this before :( `<a name="k8s-deployment">` should be `<a name="k8s-job"> to be consistent with how you defined it above.. (line 22)